pulsar-io/elastic-search/src/test/java/org/apache/pulsar/io/elasticsearch/ElasticSearchTestBase.java

```diff
@@ -39,10 +39,10 @@
 public abstract class ElasticSearchTestBase {
 
     public static final String ELASTICSEARCH_8 = Optional.ofNullable(System.getenv("ELASTICSEARCH_IMAGE_V8"))
-            .orElse("docker.elastic.co/elasticsearch/elasticsearch:8.1.0");
+            .orElse("docker.elastic.co/elasticsearch/elasticsearch:8.5.1");
```

Comment:
8.5.2 in pom.xml?

Reply:
this is the server. https://hub.docker.com/_/elasticsearch/tags 
8.5.1 is the latest 8.x